feature-libs/cart/base/root/feature-name.ts

```diff
@@ -1,2 +1,5 @@
 export const CART_BASE_FEATURE = 'cartBase';
 export const CART_BASE_CORE_FEATURE = 'cartBaseCore';
+
+export const MINI_CART = 'miniCart';
+export const ADD_TO_CART = 'addToCart';
```

Comment:
Not strongly againist having those here, but perhaps these should live with the corresponding feature.  On the other hand, this would create aloot of ooverhead for a one component entry point.